sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -521,4 +599,11 @@ public partial class RedisPersistenceSettings
         public bool? IsRdbEnabled { get { throw null; } set { } }
         public Azure.ResourceManager.RedisEnterprise.Models.PersistenceSettingRdbFrequency? RdbFrequency { get { throw null; } set { } }
     }
+    public partial class RegionSkuDetail
+    {
+        internal RegionSkuDetail() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.LocationInfo LocationInfo { get { throw null; } }
+        public string ResourceType { get { throw null; } }
```

Comment:
what kind of `type` this is? Is this a "resource type"?
Please double confirm. General guidelines do not allow the simple property name `type` and our generator will prepend `resource` when that happens therefore here we need to validate.
If it is really a resource type, does it follow the general format of an Arm resource type? something like `Microsoft.Compute/virtualMachines`? If so, please change the format of this property to `resource-type`.

Reply:
Will check with the team and confirm on this

Reply:
Probably, I'll have to check on how the ARM skus contract works exactly.

Reply:
From what I'm reading, looks like it is a resource type, but probably not the USUAL one because it won't include provider name?

"resourceType | Required, string | The resource type this object applies to. This should match the resource type name specified in the manifest. For example, for a database it would be servers/databases."

Reply:
Updated it to resourceType